src/main/kotlin/apply/domain/mission/Mission.kt

```diff
@@ -15,20 +17,24 @@ class Mission(
     @Column(nullable = false)
     val title: String,
 
-    @Column(nullable = false)
-    val description: String,
-
     @Column(nullable = false)
     val evaluationId: Long,
 
     @Embedded
     var period: MissionPeriod,
 
+    @Column(nullable = false)
```

Comment:
```suggestion
@Column(nullable = false, length = 255)
```

디폴트값이 255지만 명확하게 명시해주면 의도해서 생략한 것인지, 실수인지 덜 헷갈릴 것 같아요.

Reply:
`@Lob`으로 변경되었습니다.